docker/production/Dockerfile

```diff
@@ -21,8 +24,11 @@ ADD --chown=${USERNAME}:${GROUPNAME} . ${USER_DIR}/
 RUN pip install -r requirements.txt -r deploy_requirements.txt
 
 # Install and build the docs
+# Must pre-install to provide dependencies and version number
```

Comment:
Thanks for documenting this so we don't accidentally lose our version number again.